src/plugins/feeds/feeds-en.hbs

```diff
@@ -15,6 +15,27 @@
 
 <section>
 	<h2>Examples</h2>
+	<section class="wb-feeds limit-3" data-cors="true">
+		<h3>Canada News Centre - Manitoba</h3>
```

Comment:
Can you modify the title to indication this example use the CORS?

Reply:
Add also a small description explaining: This method fetch the ATOM feed directly and don't use a third party service.

Reply:
Great